src/main/java/org/spongepowered/common/util/SpongeHooks.java

```diff
@@ -194,6 +194,19 @@ public static void logBlockAction(final World world, @Nullable final BlockChange
         }
     }
 
+    public static void logChunkQueueLoad(final World world, final Vector3i chunkPos) {
+        if (world.isRemote) {
```

Comment:
`((WorldBridge) world).bridge$isFake()`?

Reply:
I'm using the same condition used for chunkload logging. I'd need to modify that as well

https://github.com/SpongePowered/SpongeCommon/blob/a865595b36ab95a876c8ebd46b464b23a73b09aa/src/main/java/org/spongepowered/common/util/SpongeHooks.java#L197-L199

That would disable chunkload logging for fake worlds. Is it intended ?

Reply:
Some fake worlds don't even use the normal chunk loader.